libraries/chain/controller.cpp

```diff
@@ -750,6 +750,24 @@ struct controller_impl {
          // else no checks needed since fork_db will be completely reset on replay anyway
       }
 
+      if (auto dm_logger = get_deep_mind_logger()) {
+         // FIXME: We should probably feed that from CMake directly somehow ...
+         fc_dlog(*dm_logger, "DEEP_MIND_VERSION 12");
+
+         fc_dlog(*dm_logger, "ABIDUMP START");
```

Comment:
It was suggested to use `chain_plugin#startup` to place the ABIs dump. However, the the end of startup happens too late when replaying from an existing `blocks.log` file. Maybe it should still be in `startup`, just at a better place.